cmd/brew-bundle.rb

```diff
@@ -17,7 +17,7 @@
   Usage:
   Bundler for non-Ruby dependencies from Homebrew
 
-  brew bundle            read Brewfile and install all dependencies
+  brew bundle            read Brewfile and install or upgrade all dependencies
```

Comment:
Updated in https://github.com/timurvafin/homebrew-bundle/commit/6a911be2e911cfa4bacadf6db8fcc5289f3f3db8